### DISCUSSION TITLE
Prototype Pollution in JSON5 via Parse Method

### DISCUSSION THREAD

**Discussion by iowatchGate** [npm/feedback#853] [Registry]
Within the npm package(JSON5) version 2.2.1 cannot be updated to the patched version 2.2.2 and the vulnerability is at a severity level of 7.1 in CVSS base metrics. However, on npmjs.com the search results on 'JSON5' are distributed with a new patched version of 2.2.3 instead. And this does not seem the case within npm package 'note-notifier'. Therefore my question is simple. How long until the patched 2.2.3 version comes available on Github so I can resolve our security issue? 

See the link below for more details:
https://github.com/MeewerkenOpGithub/SoestRoute/security/dependabot/76

**Comment by ljharb** [comment]
I'm not sure what you're asking. json5 v1 and v2 both have a version that addresses this CVE (which, to be clear, is not actually a vulnerability for virtually any use cases). If `note-notifier` is properly using `^`, then you can just update your lockfile - if not, then you'd need to use `overrides`, and file a bug on note-notifier's repo.
